### PR TITLE
Multi-Language Voice Contribution & Validation

### LINKED ISSUE #213: Add multilanguage support
Please, add multilanguage support. I mean made web, iOS and Android apps translatable and supporting recording for more languages besides English. For instance: Spanish, French, German, Catalan....

I'm sure these feature will be a must in near future, so It's better adding and desing them at start point.

### PR DESCRIPTION
**Pull request by Gregoor**
Fixes #213 

Some things we should figure out before merging:
- [x] #933 Shortcuts across languages
- [ ] what to do with accents across languages?

### CODE REVIEW

**Comment by danielsjf** [comment]
How many sentences are you looking for per language to start? I've added roughly 1400 Dutch sentences as well in the Github issue.

**Comment by danielsjf** [comment]
Also, when I click on get involved on a language, the pop-up doesn't fit on my cellphone screen. The last lines aren't viewable.

**Comment by jf99** [comment]
@Gregoor Nice to see this proceeding. Note however, that the 11k German sentences still contain numbers in a non-optimal [format](https://github.com/mozilla/voice-web/issues/213#issuecomment-376047665). Also, they have not been proofread yet. @Djfe please say something about this topic. If we can't get a corrected version from @Djfe until Monday, I can provide you with another > 1,000 sentences immediately. They haven't been proofread yet as well, but at least all numbers in there are written out as words.

**Comment by Djfe** [comment]
I'll try my best, it was kinda hard finding the time to read all the sentences.
Quite a lot. I'm still on it though and over 3000 sentences are proofread.
And I have lots of free time today and tomorrow and the will to finally get it done :)

**Comment by jf99** [comment]
@Djfe That's good to hear. If you can't manage to finish all sentences, I propose you post those you have finished, anyway. It won't hurt anyone if a few sentences are missing in the first week. That way we can avoid to create data with known issues.

**Comment by Gregoor** [comment]
@danielsjf Great, I think we set 2k sentences as a good starting number (cc @mikehenrty). Thanks for collecting!
RE the get-involved modal, would you mind opening an issue with reproduction steps (or in this case, device name, OS, etc.).

@jf99 @Djfe great stuff you two, thanks! I'm happy to switch those out. 3k is imo already a good start, we can add more incrementally 🙂

**Comment by jf99** [comment]
All corrections by @Djfe have been merged. We can start with all 11k sentences.
https://github.com/jf99/voice-web-de-fork/blob/german/server/data/de/11k-german-sentences.txt

**Comment by Djfe** [comment]
again: only 3k of those have been proofread.
In the other lines I only replaced the numbers.

**Comment by Gregoor** [comment]
Thanks you two! I updated the sentences with your corrections and dropped the unverified ones.

**Comment by danielsjf** [comment]
@Gregoor Here you go, 2020 sentences in Dutch. These contain the sentences from before, corrected after the comments from before (like CO2).

https://pastebin.com/AvCQYUAp

File: server/data/de/11k-german-sentences.txt [hunk 1]
```diff
@@ -0,0 +1,3000 @@
+Ich habe da mal was vorbereitet.
```
Comment (mikehenrty) [inline server/data/de/11k-german-sentences.txt]: @Gregoor, Probably want to change the name of this file to be the name of the contributor (@jf99.txt in this case)

Comment (Gregoor) [reply server/data/de/11k-german-sentences.txt]: Good point 👍 Thanks

**Pull request merged**